pkgs/top-level/all-packages.nix

```diff
@@ -3819,6 +3819,8 @@ with pkgs;
     buildGoModule = buildGo123Module;
   };
 
+  hiera-eyaml = callPackage ../tools/system/hiera-eyaml { };
```

Comment:
Not sure if it's GitHub confused with the diff, or if this has been added by mistake.

Reply:
Done